github/git_commits.go

```diff
@@ -37,6 +40,11 @@ type Commit struct {
 	// is only populated for requests that fetch GitHub data like
 	// Pulls.ListCommits, Repositories.ListCommits, etc.
 	CommentCount *int `json:"comment_count,omitempty"`
+
+	// SignKey denotes a key to sign the commit with. If not nil this key will
+	// be used to sign the commit. The private key must be present and already
+	// decrypted. Ignored if Verification.Signature is defined.
+	SignKey *openpgp.Entity
```

Comment:
nit: probably calling this `SigningKey` ?

Also, add a `json:"-"` tag to explicitly note that this does not part of the GitHub representation of this object.

Reply:
Sounds good, Updated.